internal/server/v1/page/golden/place_page/ca.Environment.json

```diff
@@ -21741,43 +21741,6 @@
             "unit": "MetricTonCO2e"
           }
         },
-        "Count_CycloneEvent": {
```

Comment:
same here

Reply:
This is the only suspicious one:

https://datacommons.org/tools/timeline#statsVar=Count_CycloneEvent&place=geoId%2F72&mh=%7B%22Count_CycloneEvent%22%3A%22784333167%22%7D&chart=%7B%22count%22%3A%7B%22pc%22%3Afalse%2C%22delta%22%3Afalse%7D%7D

vs.

https://autopush.datacommons.org/tools/timeline#statsVar=&place=geoId%2F72&mh=%7B%22Count_CycloneEvent%22%3A%22784333167%22%7D&chart=%7B%22count%22%3A%7B%22pc%22%3Afalse%2C%22delta%22%3Afalse%7D%7D